index.js

```diff
@@ -12,37 +12,459 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
   navMenu.classList.remove('active');
 }));
 
-// VALIDATE FORM
-const form = document.querySelector('#register');
-const email = document.querySelector('#email');
-const errorMessage = document.querySelector('.error-message');
-form.addEventListener('submit', (event) => {
-  if (email.value !== email.value.toLowerCase()) {
-    errorMessage.style.visibility = 'visible';
-    errorMessage.style.position = 'inherit';
-    event.preventDefault();
+// Store project information in array
+const body = document.querySelector('body');
+const projects = [
```

Comment:
- [ ]  As requested by the previous code reviewer, "`As the requirement your popup window info should relate to the project that we are opening and the data should be gotten from the array you created. We can see in your popup window only one data. Kindly manipulate the popup window data based on the array and should match the project that you opened. Thank you!`", please make sure each popup is populated with a value(object) from the project array, all the popups should not be having the same information. Each popup should be populated with a value from the projects array. Please kindly fix that.